cypress/support/index.js

```diff
@@ -19,7 +19,7 @@
 // ***********************************************************
 
 // Import commands.js using ES2015 syntax:
-import '../utils/commands';
+import { currentBackendEndpoint } from '../utils/commands';
```

Comment:
Unrelated?

Reply:
There is a lint warning in this file that we import `'../utils/commands'` twice, so I combine the two import into one.